app/views/submissions/show.html.haml

```diff
@@ -76,6 +78,14 @@
           %td= revision.pictures
           %td.text-right
             = destroy_button(revision,"Czy na pewno chcesz usunąć tę wersję wraz z recenzjami?")
+          %td
+            - if @submission.status == 'przyjęty'
+              - if revision == @submission.article_revisions.last
+                - if revision.acceptation == NIL || revision.acceptation == 'nie'
+                  = link_to "Edytuj", edit_article_revision_path(revision, submission_id: @submission.id), class: %w{btn btn-primary btn-sm}
```

Comment:
Zostawiłbym tylko tę ostatnią linijkę - w docelowej implementacji to będzie i tak wywoływane z innego miejsca.